evm_arithmetization/Cargo.toml

```diff
@@ -12,45 +12,45 @@ homepage.workspace = true
 keywords.workspace = true
 
 [dependencies]
-anyhow = "1.0.40"
+anyhow = { workspace = true }
 bytes = { workspace = true }
 env_logger = { workspace = true }
 ethereum-types = { workspace = true }
 hex = { workspace = true, optional = true }
 hex-literal = { workspace = true }
-itertools = "0.11.0"
+itertools = { workspace = true }
 keccak-hash = { workspace = true }
 log = { workspace = true }
 plonky2_maybe_rayon = { workspace = true }
 num = { workspace = true }
-num-bigint = "0.4.3"
-once_cell = "1.13.0"
-pest = "2.1.3"
-pest_derive = "2.1.0"
+num-bigint = { workspace = true }
+once_cell = { workspace = true }
+pest = { workspace = true }
+pest_derive = { workspace = true }
 plonky2 = { workspace = true }
 plonky2_util = { workspace = true }
 starky = { workspace = true }
 rand = { workspace = true }
-rand_chacha = "0.3.1"
+rand_chacha = { workspace = true }
 rlp = { workspace = true }
 rlp-derive = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
-static_assertions = "1.1.0"
-hashbrown = { version = "0.14.0" }
-tiny-keccak = "2.0.2"
+static_assertions = { workspace = true }
+hashbrown = { workspace = true }
+tiny-keccak = { workspace = true }
 serde_json = { workspace = true }
 
 # Local dependencies
-mpt_trie = { version = "0.3.0", path = "../mpt_trie" }
+mpt_trie = { workspace = true }
```

Comment:
any reason why the version is removed?

Reply:
Mistake, I'll re-add this, thanks!

Reply:
Actually yeah I was missing `version` in the root `Cargo.toml`, but here I think we can just use `workspace = true` since it will pull from that version (I think right?).

Reply:
I think it's easier for version publishing to have version numbers. But regardless, I don't understand why it would complain now for crates that were already here before?